package.json

```diff
@@ -90,10 +91,11 @@
     "webpack": "^5.72.0",
     "webpack-cli": "^4.9.2",
     "webpack-dev-server": "^4.8.1",
+    "yarn": "^1.22.19",
```

Comment:
Was this dep added, intentionally?

Reply:
Oh, no. It's caused by a typo. Let me fix it.